deploy/kubernetes/releases/csi-digitalocean-dev/snapshot-validation-webhook.yaml

```diff
@@ -20,9 +20,9 @@ webhooks:
   - name: "validation-webhook.snapshot.storage.k8s.io"
     rules:
       - apiGroups:   ["snapshot.storage.k8s.io"]
-        apiVersions: ["v1", "v1beta1"]
```

Comment:
v1beta1 was removed from webhook in v6.0.0: https://github.com/kubernetes-csi/external-snapshotter/pull/709